enhancements/observability/cluster-observability-operator-logging-stack.md

```diff
@@ -0,0 +1,405 @@
+    1. The `ClusterLogging` resource does not directly manage the Loki-based storage installation (as previously with `Elasticsearch`). Nowadays it is an installation prerequisite, by the user, to create a `LokiStack` resource.
+    2. The OpenShift Console visualization plugin namely [logging-view-plugin](logging-view-plugin) is not installed/upgraded by any resource. The user can **only** opt-in to use it when installing CLO.
+
+Therefore this proposal aims to address the challenges users face when they follow the second theme, i.e. using OpenShift Logging as an all-in-one in-cluster offering.
+
+## Motivation
+
+Red Hat Observability products followed similar evolution paths as OpenShift Logging by gradually adding support for a variety of customer environments: on-premises, OpenShift Dedicated and Managed Services.
+
+Even though each product's development lifecycle is and should stay independent the sheer amount of installation/upgrading procedures impose a high tax on users. Auxiliary components that unify user experience under the hood of the OpenShift Console as well as emerging ones like correlation amplifies this trend.
+
+As a result, to enhance user experience with OpenShift Observability products this proposal aims to be a foundation for future observability enhancements on:
+
+* How to declaratively install/upgrade each Observability product without applying separate procedures each time using COO only.
+* How to uniformly handle the lifecycle, version skew and OpenShift Console dependencies of all Observability view plugins.
+* How to add more value across signals applying automation (i.e. enable correlation components, sharing scheduling primitives, labeling rules/alerts).
+
+Notwithstanding this proposal is limited to accomplish the above for the OpenShift Logging product only.
+
+![coo-vision](./assets/coo-vision.png)
+
+### User Stories
+
+As an OpenShift administrator, I want to install the logging capability creating a single custom resource, so I can easily set up custom-tailored logging for infrastructure and workloads.
+
+As an OpenShift administrator, I want to upgrade the logging capability editing a custom resource, so I can focus only on post-upgrade steps.
+
+As an OpenShift administrator, I want to edit loggging components through underlying custom resoures, so I can improve overall reliability and performance of individual components when needed.
+
+As an OpenShift administrator, I want to use metrics to logs correlation without additional installation procedure, so I can troubleshoot infrastructure/workloads by navigating from metrics to logs and vice versa.
```

Comment:
is this "infrastructure workloads" or are you saying infrastructure and/or workloads?  trying to distinguish them as two things?